cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingTerminalMapper.java

```diff
@@ -399,6 +401,17 @@ private Optional<Terminal> findBusBranch() {
         }
     }
 
+    static Optional<Terminal> findEquivalentTerminalForVoltageRegulatingTerminalDefinedAtBusbarSection(String cgmesTerminalId, Context context) {
+        CgmesTerminal busbarSectionCgmesTerminal = context.cgmes().terminal(cgmesTerminalId);
+        if (busbarSectionCgmesTerminal == null) {
+            return Optional.empty();
+        }
+        Bus busBreakerView = context.network().getBusBreakerView().getBus(busbarSectionCgmesTerminal.topologicalNode());
+        return busBreakerView != null
+                ? EquivalentTerminalFinderVoltageControl.best(EquivalentTerminalFinderVoltageControl.findTerminalsBusBranch(busBreakerView.getVoltageLevel(), busBreakerView))
```

Comment:
not sure here, my understanding is that  `findTerminalsBusBranch` does not consider opened switches ?

Reply:
Ok understood.